clang/docs/Modules.rst

```diff
@@ -421,13 +421,7 @@ As an example, the module map file for the C standard library might look a bit l
 
 .. parsed-literal::
 
-  module std [system] [extern_c] {
```

Comment:
Why remove the extern_c here? This seems like a pretty good example of a time when it'd make sense to use this attribute.

Reply:
I see you mentioned this in the PR description. I suppose whether this is appropriate depends on whether these headers are all pure C headers or if they themselves try to support inclusion from C++.  I don't really mind whether we keep this or remove it.